src/settings_gui.cpp

```diff
@@ -441,6 +462,21 @@ struct GameOptionsWindow : Window {
 				this->SetDirty();
 				break;
 
+			case WID_GO_BASE_SFX_VOLUME:
+			case WID_GO_BASE_MUSIC_VOLUME: {
+				extern bool ClickVolumeSliderWidget(Rect r, Point pt, byte & value);
```

Comment:
```suggestion
				extern bool ClickVolumeSliderWidget(Rect r, Point pt, byte &value);
```
(assuming no include change)